assets/less/content.less

```diff
@@ -1,203 +1,238 @@
-  padding-bottom: 10px;
-}
-
-/* OTHERS */
-
-div.breadcrumbs {
-  padding: 3px 15px 3px 16px;
-  background-color: #EEE;
-  margin: 0 -26px 10px -18px;
-  line-height: 28px;
-  display: block;
-}
-
-/* SUMMARY LINKS */
-
-ul.summary-links {
-  margin: 0 0 1em 0;
-  padding: 0;
-}
-
-ul.summary-links li {
-  display: inline-block;
-  list-style-type: none;
-  width: 7em;
-  text-align: center;
-  background: #f0f0f0;
-}
-
-/* EXTERNAL LINKS */
-#content a.external span {
-  padding-left: 3px;
-}
-
-/* DOCSTRING */
-
-section.docstring,
-p.docstring {
-  margin-right: 0;
-}
-
-.docstring h1,
-.docstring h2,
-.docstring h3,
-.docstring h4 {
-  border: 0;
-  margin: 1.8em 0 0.5em;
-}
-
-.docstring h3,
-.docstring h4 {
-  font-size: 1.2em;
-}
-
-.docstring ul {
-  padding-left: 20px;
-}
-
-.docstring li > p {
-  margin: 0;
-}
-
-/* SUMMARY */
-
-div.detail-header-links {
-  float: right;
-  padding-right: 10px;
-}
-
-a.to-top-link {
-  padding-left: 0.3em;
-  font-size: 1em;
-  font-weight: normal;
-}
-
-.summary {
-  padding: 0;
-  width: 100%;
-}
-
-table.summary tr:nth-child(odd) {
-  background: #f0f0f0;
-}
-
-table.summary tr:nth-child(even) {
-  background: #fafafa;
-}
-
-table.summary tr td {
-  border: 0;
-  padding-top: 0.5em;
-  padding-bottom: 0.5em;
-}
-
-td.summary-signature {
-  padding-right: 0.5em;
-}
-
-td.summary-synopsis {
-  padding-left: 0.5em;
-  width: 90%;
-}
-
-td.summary-synopsis p {
-  margin: 0;
-}
-
-/* DETAILS LIST */
-
-.spec,
-.typespec {
-  font: bold 1em Courier, monospace;
-}
-
-ul.spec {
-  padding: 6px 10px 6px 25px;
-  list-style-type: none;
-}
-
-.type-detail {
-  margin: 20px 0 0 0;
-}
-
-.type-detail > div.typespec-doc {
-  margin-left: 10px;
-}
-
-.type-detail > div.typespec-doc > p {
-  margin: 5px 0;
-}
-
-.detail {
-  margin-top: 15px;
-  padding-top: 0;
-}
-
-.detail:nth-child(2) {
-  border: 0;
-}
-
-div.detail-header {
-  float: left;
-  display: block;
-  width: 100%;
-  margin-bottom: 10px;
-}
-
-span.signature {
-  font-family: @monoFontFamily;
-  font-size: 15px;
-  padding-left: 10px;
-
-  strong {
-    font-weight: 400;
-  }
-}
-
-
-span.detail-type {
-padding-left: 10px;
-  font-size: 14px;
-  font-family: @sansFontFamily;
-  font-weight: 300;
-}
-
-a.detail-link {
-  padding-left: 0.3em;
-}
-
-.detail-header-links > a {
-  border: none;
-  font-size: 14px;
-}
+  font-size: 18px;
+  line-height: @baseLineHeight;
+  padding: 20px 60px;
+  margin-top: 70px;
+
+  h1 {
+    font-family: @sansFontFamily;
+    font-size: @h1FontSize;
+    line-height: @h1LineHeight;
+    margin-top: @h1MarginTop;
+    margin-bottom: @h1MarginBottom;
+
+    font-weight: 700;
+    padding-bottom: @h1Line;
+    border-bottom: 1px solid @lightGray;
+  }
+
+  h1 small {
+    color: #888;
+  }
+
+  h2 {
+    font-family: @sansFontFamily;
+    font-size: @h2FontSize;
+    line-height: @h2LineHeight;
+    margin-top: @h2MarginTop;
+    margin-bottom: @h2MarginBottom;
+    font-weight: 700;
+  }
+
+  h1.section-heading {
```

Comment:
This can be nested, right?

```css
h1 { 
  font-family: @sansFontFamily;
  font-size: @h1FontSize;
  line-height: @h1LineHeight;
  margin-top: @h1MarginTop;
  margin-bottom: @h1MarginBottom;
  font-weight: 700;
  padding-bottom: @h1Line;
  border-bottom: 1px solid @lightGray;

  .section-heading {
    color: @green;
    font-family: @sansFontFamily;
    font-weight: normal;
    text-transform: uppercase;
    font-size: @h3FontSize;
    line-height: @h3LineHeight;
    margin-top: @h3MarginTop;
    margin-bottom: @h3MarginBottom;
    border: none;
  }
}
```